app/tests/unit/forms/email/reminderService.spec.js

```diff
@@ -27,6 +27,16 @@ const schedule = {
   closeSubmissionDateTime: null,
 };
 
+describe('_init', () => {
```

Comment:
Although this looks like I'm testing an internal function, it's just misnamed. It is neither internal, nor does it do initialization - this function actually sends out the reminders.